app/Console/Commands/UpdateArticleViewCounts.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Article;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Http;
+
+final class UpdateArticleViewCounts extends Command
+{
+    protected $signature = 'lio:update-article-view-counts';
+
+    protected $description = 'Queries the Fathom Analytics API to update the view counts for all articles';
+
+    protected $siteId;
+
+    protected $token;
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->siteId = config('services.fathom.site_id');
+        $this->token = config('services.fathom.token');
+    }
+
+    public function handle()
+    {
+        if (! $this->siteId || ! $this->token) {
+            $this->error('Fathom site ID and token must be configured');
+
+            return;
+        }
+
+        Article::published()->chunk(100, function ($articles) {
```

Comment:
I suggest adding `recent()` method to give the most recent articles more priority for updating views

Reply:
This indeed isn't necessary.